spec/index.bs

```diff
@@ -4255,28 +4259,29 @@ Issue: define this as an algorithm with (texture, mipmapLevel) parameters and us
 
     1. Fail if the following conditions are not satisfied:
         <div class=validusage>
-            - If |heightInBlocks| &gt; 1 or |copyExtent|.[=Extent3D/depth=] &gt; 1:
-                - |layout|.{{GPUTextureDataLayout/bytesPerRow}} must be
-                    greater than or equal to the number of |bytesInACompleteRow|.
-            - If |copyExtent|.[=Extent3D/depth=] &gt; 1 or
-                |layout|.{{GPUTextureDataLayout/rowsPerImage}} &gt; 0:
-                - |layout|.{{GPUTextureDataLayout/rowsPerImage}} must be
-                    greater than or equal to |heightInBlocks|.
-
-            These ensure the copy's subregions of the linear data do not overlap.
+            - If specified, |layout|.{{GPUTextureDataLayout/bytesPerRow}}
+                must be greater than or equal to |bytesInACompleteRow|.
+            - If specified, |layout|.{{GPUTextureDataLayout/rowsPerImage}}
+                must be greater than or equal to |heightInBlocks|.
+            - If |heightInBlocks| &gt; 1,
```

Comment:
nit: I'd put the validation that something must be specified before the validation that if it is specified it must be more than X